### PR TITLE
Rewrite the Ivy revisions section

### PR DESCRIPTION
**Pull request by eed3si9n**
Ref https://github.com/sbt/website/pull/1203

### CODE REVIEW

File: src/reference/guide/08-Library-Dependencies.md [hunk 1]
```diff
@@ -153,32 +153,26 @@ to ensure binary compatibility.
...
+Cousier and Ivy can select the latest revision of a module according to constraints you specify. 
+Instead of a fixed revision like
+`"1.6.1"`, you specify `"latest.integration"`, `"2.9.+"`, or `"[1.0,)"`. 
+These dynamic versions might work one day, but break the next, and make the build non-repeatable.
+
+A Maven version range such as `[1.3.0,)` can be used to specify a dependency,
+and it can appear transitively in existing POM files. When a dependency manager
+like Coursier finds a version range, it will go out to the Internet to find the latest version.
+This could lead to surprising behavior where the effective version changes over time,
+even when there is a specified version of the library within the version range.
```
Comment (wenhoujx) [inline src/reference/guide/08-Library-Dependencies.md]: im still confused about the referenced `specified version`, if a range was provided as `version`, where did the `specified version` come from?

Comment (eed3si9n) [reply src/reference/guide/08-Library-Dependencies.md]: I guess transitively from other subproject or libraries.

Comment (wenhoujx) [reply src/reference/guide/08-Library-Dependencies.md]: ok, mind to make that into the rewrite? i think that might help others who think like me 😝 
just a nit

**Review by wenhoujx — APPROVED** [review]
lgtm, but i don't have power.

File: src/reference/guide/08-Library-Dependencies.md (hunk 1 above)
Comment (eed3si9n) [reply src/reference/guide/08-Library-Dependencies.md]: Addressed in 55ef0a5.

**Pull request merged**